### DISCUSSION TITLE
Separating Conductor UI into a separate repository

### DISCUSSION THREAD

**Discussion by peterlau** [Netflix/conductor#3282] [Announcements]
### Announcement: Separating Conductor UI into a separate repository

In order to improve the maintainability of the UI code, and isolate UI changes from the backend release cycle, we have decided to migrate the UI code into its own repository. This repository can be found at https://github.com/Netflix/conductor-ui.

For backward compatibility and to protect the integrity of existing 3.x builds, the `/ui` folder will remain in the main `Netflix/conductor` repo for the life of the 3.x release. However it will receive no other updates beyond security updates. 

Moving forward, features and improvements will be contributed exclusively to the `Netflix/conductor-ui` repository. Discussion, Issues and Pull Requests should also be directed to the relevant sections under that repo.

Over the coming weeks we will be updating the builds in `/docker` to take advantage of the separate UI repository location.

**Comment by saradhi-atluri** [comment]
> 

Could you please share the docker UI repository location if it is available?
